Data_Dictionary.md

```diff
@@ -211,4 +212,44 @@ LAMP calculated dataset containing planned `route_id` and `service_id` combinati
 | trip_id | string | `trip_id` from [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)
 | direction_id | int8 | `direction_id` from [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)
 | block_id | string | `block_id` from [trips.txt](https://gtfs.org/schedule/reference/#tripstxt)
-| static_version_key | int64 | key used to link GTFS static schedule versions between tables |
\ No newline at end of file
+| static_version_key | int64 | key used to link GTFS static schedule versions between tables |
+
+
+# GTFS Alerts Archive
+
+Each row represents an update to an Realtime Alert, paired with unique Route, Stop, and Direction information. All timestamp fields are in POSIX time, the integer number of seconds since 1 January 1970 00:00:00 UTC. All datetimes in are Eastern Standard Time.
+
+| field name | type | description |
+| ---------- | ---- | ----------- |
```

Comment:
Should probably include a `source` column if any fields would be considered a LAMP created aggregation